src/codec_dav1d.c

```diff
@@ -6,7 +6,14 @@
 #if defined(_MSC_VER)
 #pragma warning(disable : 4201) // nonstandard extension used: nameless struct/union
```

Comment:
Ideally the _MSC_VER warning suppression should also be done with push and pop. I didn't do that in this CL.